.gitignore

```diff
@@ -10,3 +10,5 @@ public/bundle.js.map
 
 # NPM errors
 npm-debug.log
+
+secrets
```

Comment:
You need to change this to "secrets.js" to target that file- right now it's looking for a folder called "secrets"